app/ide-desktop/lib/client/src/index.js

```diff
@@ -26,7 +26,7 @@ const resources = path.join(root, '..')
 const project_manager_executable = path.join(
     resources,
     project_manager_bundle,
-    PROJECT_MANAGER_IN_BUNDLE_PATH
+    PROJECT_MANAGER_IN_BUNDLE_PATH // This is a define.
```

Comment:
I dont understand this comment.

Reply:
Reworded. I was confused as to where this value comes from, when I first encountered a define.